hazelcast/src/test/java/com/hazelcast/internal/partition/membergroup/MemberGroupFactoryTest.java

```diff
@@ -202,14 +197,45 @@ public void testNodeMetadataAwareMemberGroupFactoryCreateMemberGroups() {
     public void testNodeAwareMemberGroupFactoryThrowsIllegalArgumentExceptionWhenNoMetadataIsProvided() {
         MemberGroupFactory groupFactory = new NodeAwareMemberGroupFactory();
         Collection<Member> members = createMembersWithNoMetadata();
+        groupFactory.createMemberGroups(members);
+    }
+
+    @Test
+    public void testPlacementMetadataAwareMemberGroupFactoryCreateMemberGroups() {
+        MemberGroupFactory groupFactory = new PlacementAwareMemberGroupFactory();
+        Collection<Member> members = createMembersWithPlacementAwareMetadata();
         Collection<MemberGroup> memberGroups = groupFactory.createMemberGroups(members);
 
-        assertEquals("Member Groups: " + String.valueOf(memberGroups), 3, memberGroups.size());
+        assertEquals("Member Groups: " + memberGroups, 3, memberGroups.size());
         for (MemberGroup memberGroup : memberGroups) {
-            assertEquals("Member Group: " + String.valueOf(memberGroup), 1, memberGroup.size());
+            assertEquals("Member Group: " + memberGroup, 1, memberGroup.size());
         }
     }
 
+    private Collection<Member> createMembersWithPlacementAwareMetadata() {
+        Collection<Member> members = new HashSet<>();
+        MemberImpl member1 = new MemberImpl(new Address("192.192.0.1", fakeAddress, 5701), VERSION, true);
+        member1.setAttribute(PartitionGroupMetaData.PARTITION_GROUP_PLACEMENT, "us-east-1a-placement@1");
```

Comment:
The '@' notation here is also used by AWS discovery. Does it cause a conflict with any other internal notation for partitioning - for instance https://docs.hazelcast.org/docs/latest/manual/html-single/#controlling-partitions?

Reply:
Hmm, maybe it's better to just use `-` instead of `@`. Wdyt?

Reply:
Any char could actually go there. The motivation was making it somewhat similar to the Hazelcast notation - not necessary at all. Dash seems also OK.

Reply:
This placement group string is taken as is (no tokenization, etc) and used as a key in a simple `HashMap<String, MemberGroup>`. I think in general `@` may suggest the value is structured in some way. In that sense I think `-` is better, but no strong opinion on this.

Reply:
Fair enough. Will change the notation on AWS PR as well.